app/router.ts

```diff
@@ -54,7 +54,9 @@ Router.map(function() {
         this.mount('registries', { path: '--registries' });
     }
 
-    this.route('guid-file', { path: '--file/:guid' });
+    this.route('guid-file', { path: '--file/:guid' }, function() {
+        this.route('metadata');
+    });
```

Comment:
This definitely shouldn't be here. This route doesn't exist anymore.

Reply:
Same as above. Would you like all of it removed?

Reply:
Yes, please.

Reply:
Same here.